roulette/src/App.js

```diff
@@ -0,0 +1,32 @@
+import './App.css';
+import './buttons.css';
+import Add from './components/addMovie/Add';
+import MovieList from './components/movieList/MovieList';
+import MovieFilter from './components/movieFilter/MovieFilter';
+import MovieSearch from './components/movieSearch/MovieSearch';
+import movieData from './data/movies';
+import ErrorBoundary from './components/movieError/ErrorBoundary';
+import MovieSort from './components/movieSort/MovieSort';
```

Comment:
Consider grouping imports of the same level together (./oneThing and ../anotherThing are different levels) and separating each level with an empty line. Additionally, order imports top-down from most-global to most-local - it's usually:
- public library imports
- private library imports
- imports of different levels within the project (level ~= the number of "../")
- local imports for thing that are in the same folder as current file)
E.g. https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md

Additionally, consider adding this plugin to your project to automate that.